libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -188,6 +188,10 @@
 #define AP_EXTENDED_DSHOT_TELEM_V2_ENABLED HAL_REQUIRES_BDSHOT_SUPPORT
 #endif
 
+#ifndef AP_EXTENDED_ESC_TELEM_ENABLE
+#define AP_EXTENDED_ESC_TELEM_ENABLE 0
```

Comment:
this should at least be enabled on SITL so it is tested in CI

Reply:
probably should be in our build options script so it can be enabled via custom build server

Reply:
made changes in AP_HAL_Boards.h, build_options.py and extract_features.py